BUILD

```diff
@@ -2053,6 +2053,26 @@ grpc_cc_library(
     alwayslink = 1,
 )
 
+grpc_cc_library(
+    name = "grpc++_channelz",
```

Comment:
Since the public header is under `include/grpcpp/ext/`, maybe rename it to `grpcpp_channelz`?

Reply:
Done (changes the proto lib too for consistancy)